### DISCUSSION TITLE
Fixing the first rows in the table

### DISCUSSION THREAD

**Discussion by rafael-borges-garcia** [ant-design/ant-design#49509] [Q&A]
Hi all,

I'm using Ant Table component in a huge project and I'm stuck on something I can't solve  😢  

Is there any way I can fix the first two rows of a table?
Without using Summary (I'm already using Summary for the footer).
As in excel. I can freeze some rows and leave them fixed at the top.

Thanks 😄

**Comment by zxlvera** [comment]
Similar [question](https://github.com/ant-design/ant-design/discussions/37950), perhaps you could try this out
